### ISSUE TITLE
Testing on new dataset

### ISSUE THREAD

**Issue by assafbot** [google-deepmind/open_x_embodiment#9]
Hi,

Can you provide instructions on how to create a new dataset to use with the pretrained model?

According to the paper, the model can support either absolute and relative positions as well as velocities.
How is this implemented/supported by the model/dataset?

Do I need to compensate for the different viewpoints in each dataset in any way?

Thanks,

**Comment by kpertsch** [comment]
Hi!
You can follow the instructions [here](https://github.com/kpertsch/rlds_dataset_builder) to parse your custom dataset into the RLDS format used by RT-X.

For the actions, it's best to express them in *some* end-effector format of 3D position deltas and 3D euler angle rotation deltas. You don't need to compensate for different viewpoints, we did not align the viewpoints / coordinate frames in the RT-X training datasets either.

**Comment by assafbot** [comment]
Thanks,
In the paper, you mentioned that the actions are normalized.
Is there any example of how this is implemented?

**Comment by kpertsch** [comment]
@quanvuong could you help out with details on how the actions are normalized for RT-X training?

**Comment by quanvuong** [comment]
We are planning to release the Jax version of the RT-1-X model, which will includes code that normalizes the action during training.
